src/test/java/com/iexec/standalone/compute/pre/PreComputeServiceTests.java

```diff
@@ -419,5 +430,12 @@ void shouldAllTeeSessionGenerationErrorHaveMatch() {
                     .isNotNull();
         }
     }
+
+    @Test
+    void shouldFailedAndReturnNull() {
+        TeeSessionGenerationError error = mock(TeeSessionGenerationError.class);
+        when(error.name()).thenReturn("NA");
+        Assertions.assertThat(preComputeService.teeSessionGenerationErrorToReplicateStatusCause(error)).isNull();
```

Comment:
For consistency with other updates in same class ?
```suggestion
        assertThat(preComputeService.teeSessionGenerationErrorToReplicateStatusCause(error)).isNull();
```